docs/ref/pgcopydb_clone.rst

```diff
@@ -729,6 +729,16 @@ PGCOPYDB_SPLIT_TABLES_LARGER_THAN
    When ``--split-tables-larger-than`` is ommitted from the command line,
    then this environment variable is used.
 
+PGCOPYDB_ESTIMATE_TABLE_SIZES
```

Comment:
Actual documentation coverage for the option is missing, see the option list at https://pgcopydb.readthedocs.io/en/latest/ref/pgcopydb_clone.html#options. In particular, we need to document the `vacuumdb` call that is implemented when using that option, as that's not transparent to the user at all.